src/lib/database/migrations/1616168146993-V39_RenamePropertiesToSnakeCase.ts

```diff
@@ -0,0 +1,99 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class V39RenamePropertiesToSnakeCase1616168146993 implements MigrationInterface {
```

Comment:
```suggestion
export class V39RenamePropertiesToKebabCase1616168146993 implements MigrationInterface {
```

Also update file name and reference in `SetMigrations` accordingly